src/FastRouteRouter.php

```diff
@@ -11,6 +11,7 @@
 
 use FastRoute\DataGenerator\GroupCountBased as RouteGenerator;
 use FastRoute\Dispatcher\GroupCountBased as Dispatcher;
+use FastRoute\Dispatcher as DispatcherInterface;
```

Comment:
I think it would be clearer if we remove alias on `FastRoute\Dispatcher\GroupCountBased` and use it directly in `createDispatcherCallback`. Then we can use `Dispatcher` here and constants like `FOUND`, `METHOD_NOT_ALLOWED`, `NOT_FOUND` are going to be used from the interface (as it should be).

Reply:
Done! :+1: